### PR TITLE
Assignment 2 section 3.2

### PR DESCRIPTION
**Pull request by dlarocque**

### CODE REVIEW

File: software-requirement-specification.md [hunk 1]
```diff
@@ -99,29 +106,37 @@ This section describes specific features of the software project. Specify the re
...
+- **Data Encryption:** All calls must be end-to-end encrypted.
+- **Authentication:** Multi-factor authentication must be implemented for both Administrators and regular users.
+- **Session Timeout:** Administrator sessions in the User Interface should automatically timeout after 30 minutes of inactivity.
 
 #### 3.2.5 Interfaces
 
+- **User Interface:** The UI must be intuitive and require no more than one hour of training for efficient utilization.
+- **Onboarding Time:** New users should be able to complete the onboarding process within 20 minutes, on average.
+- **Accessibility:** The system should be usable across desktop computers, smartphones, tablets, and other devices.
+
```
Comment (BarbzCodez) [inline software-requirement-specification.md]: This part was 'technically' cody's but he's pr is a bit different so we'll see

Comment (dlarocque) [reply software-requirement-specification.md]: Was this not supposed to be under section 3.2?

File: software-requirement-specification.md [hunk 2]
```diff
@@ -99,29 +106,37 @@ This section describes specific features of the software project. Specify the re
...
-
-This section describes non-functional features of the software project. Specify the requirements as user stories.
+This section outlines the non-functional requirements crucial for the U↑ internet-based phone system, focusing on performance, reliability, availability, security, and interfaces.
 
 #### 3.2.1 Performance
 
-- The product shall take initial load time depending on internet connection strength which also depends on the media from which the product is run.
+- **System Latency:** Call rerouting should happen within a 5-second window.
+- **Concurrency:** The system must support at least 500 concurrent calls without degradation in performance.
+- **Scalability:** The system should accommodate a 20% increase in user devices annually without requiring a system overhaul.
```
Comment (CodyWallbridge) [inline software-requirement-specification.md:152]: might be worth adding "- The performance shall depend upon hardware components of the client/customer."

File: software-requirement-specification.md [hunk 3]
```diff
@@ -99,29 +106,37 @@ This section describes specific features of the software project. Specify the re
...
 
-- The system shall use secure sockets in all transactions that include any confidential customer information.
-- The system shall automatically log out all customers after a period of inactivity.
+- **Data Encryption:** All calls must be end-to-end encrypted.
+- **Authentication:** Multi-factor authentication must be implemented for both Administrators and regular users.
+- **Session Timeout:** Administrator sessions in the User Interface should automatically timeout after 30 minutes of inactivity.
 
 #### 3.2.5 Interfaces
 
+- **User Interface:** The UI must be intuitive and require no more than one hour of training for efficient utilization.
```
Comment (CodyWallbridge) [inline software-requirement-specification.md:173]: might be worth including some of the hardware interface, user interfaces, software interface and communications sections from the example doc

File: software-requirement-specification.md (hunk 2 above)
Comment (dlarocque) [reply software-requirement-specification.md:152]: I agree that this should go in the doc, but in my opinion it's not a non-functional requirement

is there a way I could rephrase it to make it a requirement, or is there a better place in the doc for this?

**Review by BarbzCodez — APPROVED** [review]
lgtm :)
